test/Microsoft.ML.Tests/TrainerEstimators/MatrixFactorizationTests.cs

```diff
@@ -485,10 +485,14 @@ public void OneClassMatrixFactorizationInMemoryDataZeroBaseIndex()
             var testPrediction = model.Transform(testDataView);
 
             var testResults = mlContext.Data.CreateEnumerable<OneClassMatrixElementZeroBasedForScore>(testPrediction, false).ToList();
+
+            // REVIEW: We are seeing lower precision on non-Windows platforms
```

Comment:
>REVIEW [](start = 15, length = 6)

maybe use below tags in comments so we can find this easily from task list: 
TODO TEST_STABILITY: